Sources/PandaSDK/ConfiguredPanda.swift

```diff
@@ -551,6 +551,14 @@ final public class Panda: PandaProtocol, ObserverSupport {
         viewModel.onSupportUkraineAnyButtonTap = { [weak self] in
             self?.send(event: .onSupportUkraineAnyButtonTap)
         }
+        
+        viewModel.onDontHaveApplePay = { [weak self]  screenId, destination in
```

Comment:
Remove spacing and fix ID naming
```suggestion
        viewModel.onDontHaveApplePay = { [weak self] screenID, destination in
```